### ISSUE TITLE
Method not found when using Hedgehog 0.12

### ISSUE THREAD

**Issue by AlexeyRaga** [dharmaturtle/fsharp-hedgehog-xunit#6]
This example fails when using with `Hedgehog 0.12.x` but runs fine with `Hedgehog 0.11.x`

```fsharp
[<Property>]
let ``should be ok``(value : int) =
    value = value
```

Here is the exception:

```
System.MissingMethodException
Method not found: 'Int32 Hedgehog.FailureData.get_Size()'.
   at InternalLogic.tryRaise(Report report)
   at XunitOverrides.PropertyTestInvoker.CallTestMethod(Object testClassInstance) in /home/runner/work/fsharp-hedgehog-xunit/fsharp-hedgehog-xunit/src/Hedgehog.Xunit/XunitOverrides.fs:line 11
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<<InvokeTestMethodAsync>b__1>d.MoveNext() in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 257
```

I checked with using `hedgehog-experimental` and `GenX.auto` directly and it works, so the bug must be somewhere in this library.

The following dependencies are used:

```
"Hedgehog" Version="0.12.1"
"Hedgehog.Xunit" Version="0.3.0"
"xunit" Version="2.4.1"
```

**Comment by dharmaturtle** [comment]
I believe this issue should be resolved as of [the latest commit](https://github.com/dharmaturtle/fsharp-hedgehog-xunit/commit/716a51bce8d47529dd6a8728828acaa22a947303). I'm not cutting a release yet though because I'm not quite happy some details regarding `recheck`. Need to think about this a bit more >_>

**Comment by dharmaturtle** [comment]
Fixed with [the `0.4.0` release](https://www.nuget.org/packages/Hedgehog.Xunit/0.4.0). The recheck thing I mentioned turned out to be [an upstream issue](https://github.com/hedgehogqa/fsharp-hedgehog/issues/408). I'll look into that Soon(TM).
